library/HTMLPurifier/Lexer/PH5P.php

```diff
@@ -1574,7 +1574,7 @@ private function EOF()
     }
 }
 
-class HTML5TreeConstructer
```

Comment:
**Suggestion**: Over a rename of a class which without final/@internal, others could be using, extend and then mark this one as deprecated + a TODO for a major version.

Reply:
How to do that?

Reply:
Done 🍏

Reply:
Sorry to be clear, this is also probably wanted in the other `class HTML5TreeConstructer` in `maintenance/PH5P.php`, else this would also be a breaking change which is best in a major version.

Reply:
Also Done.

This is the point I could work on this PR. Please continue yourselves.
For me this is random synthetic `typos` exercise, I've never seen htmlpurifier before.